src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  let result = sourceString.split(';').map((arr) => arr.split(':'));
+
+  result = result.map(
+    (subArray) => subArray.map((string) => string.trim()),
+    // eslint-disable-next-line prettier/prettier
+  );
```

Comment:
don't disable  prettier.
```suggestion
  result = result.map((subArray) => (
    subArray.map((string) => string.trim())
  ));
```
u can change ur syntax